clients/ranchercli/download.go

```diff
@@ -0,0 +1,92 @@
+package ranchercli
+
+import (
```

Comment:
Hey Markus why is this called download.go?

Reply:
It was initially because it was just setting up the Rancher CLI, but it's a bit outdated now. It can be renamed to something else, I don't have a preference as to exact name

Reply:
I'll rename to `client.go`.